src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -190,7 +196,18 @@ public function getComments($id)
             $this->client->api('issues')->show($this->getCurrentProject()->id, $id)
         );
 
-        return $issue->showComments();
+        $comments = [];
+        $issue->showComments();
```

Comment:
Can't you access the data form the `getData` method on the model?
E.G

```
$comments = array_map(function(Note $comment) {
    $data = $comment->getData();

    return [
        'id' => $data['id'],
        'user' => ['login' => $data['author']['username']],
        'body' => $data['body'],
        'created_at' => new \DateTime($data['created_at']),
    ];
}, $issue->showComments());
```

(Note: I haven't tested this, but looking at the gitlab api code, you should be able to get all the values from the `getData` method)